src/main/java/teammates/ui/template/ResponseRow.java

```diff
@@ -0,0 +1,38 @@
+package teammates.ui.template;
+
+import java.util.List;
+
+public class ResponseRow {
+    private String giverName;
+    private String recipientName;
+    private String response;
+    private List<FeedbackResponseCommentRow> feedbackResponseCommentRows;
+    
+    public ResponseRow(String giverName, String recipientName, 
+                           String response, List<FeedbackResponseCommentRow> feedbackResponseCommentRows) {
+        this.giverName = giverName;
+        this.recipientName = recipientName;
+        this.response = response;
+        this.feedbackResponseCommentRows = feedbackResponseCommentRows;
+    }
+    
+    public String getGiverName() {
+        return giverName;
+    }
+    
+    public String getRecipientName() {
+        return recipientName;
+    }
+    
+    public String getResponse() {
+        return response;
+    }
+    
+    public List<FeedbackResponseCommentRow> getFeedbackResponseCommentRows() {
+        return feedbackResponseCommentRows;
+    }
+    
+    public boolean isCommentsEmpty() {
+        return feedbackResponseCommentRows == null || feedbackResponseCommentRows.size() <= 0;
+    }
```

Comment:
Is this needed?
If it's called in the tags, use `empty`.
If it's called in the tests, just `assertEquals` with `null` or `assertTrue` with `isEmpty()`.